Scheduler/v6/_summit.yaml

```diff
@@ -52,16 +54,18 @@ maintel:
         efd_columns:
           - fwhm
         efd_delta_time: 300.0
-        fill_value: 1.0
+        fill_value: null
```

Comment:
Is null acceptable here? I think we left it at 1.0 in other cases.

Reply:
For the record, I favor null over 1.0 so at least we have an indication that a fill value is being used, just want to be sure null doesn't affect anything downstream.